modules/client/src/tokenVoting/internal/client/methods.ts

```diff
@@ -328,6 +344,9 @@ export class TokenVotingClientMethods extends ClientCore
     if (!isProposalId(proposalId)) {
       throw new InvalidProposalIdError();
     }
+    if (!/^0x[A-Fa-f0-9]{40}_(0x[A-Fa-f0-9]{1,64})$/.test(proposalId)) {
```

Comment:
Same here, use the common `isProposalId`

Reply:
same